build.sh

```diff
@@ -1,5 +1,9 @@
 #!/bin/sh
 
+# Build ghz Docker image.
+# See ghz-tool/Dockerfile for details/version
+docker build -t ghz_bench:latest ./ghz-tool/
```

Comment:
We should update the cleanup script at line 6 with the `ghz` image name

Reply:
Fixed on 196e4b6.